pyanaconda/bootloader.py

```diff
@@ -2320,7 +2320,7 @@ class EXTLINUX(BootLoader):
     _config_dir = "/boot/extlinux"
 
     # stage1 device requirements
-    stage1_device_types = ["disk"]
+    _boot_stage1_device_types = ["disk"]
```

Comment:
How do you know this should be the correct name? I can't see it used anywhere.

Reply:
You are right, the attribute should be removed. Fixed.